js/hoot/control/utilities/ImportDirectory.js

```diff
@@ -880,6 +946,38 @@ Hoot.control.utilities.importdirectory = function(context) {
         return true;
     };
 
+    /**
+    * @desc Function to append FCode Description based on FCODE
+    * @desc Assumes that layer name is a valid FCODE
+    * @oaram fcodeList - JSON of FCODEs and Descriptions
+    * @param filesList - Selected files list.
+    **/
+    var appendFCodeDescription = function(fcodeList) {
+        var filesList = _getFilesList();
+        _.each(filesList, function(f){
```

Comment:
think it might be best to add a class to the options and then run a selectAll on that class. Will implement this change.